README.md

```diff
@@ -0,0 +1,36 @@
+# Hack with docker
+
+To quickly start hacking, hm-admin comes with a docker ready development environment.
+
+It leverages three services:
+
+- `hm-admin`: The interesting part in our case. Hm-admin is running in this container and hm-admin sources are shared with the host
+- `api` and `mysql`: Those containers are responsible for the API to be developed and tested against. The API is made from https://api-platform.com/.
```

Comment:
It's incorrect, if you want to stick with from you have to change the url the this api-platform/core github.